modules/installation-gcp-user-infra-wait-for-bootstrap.adoc

```diff
@@ -36,7 +36,7 @@ stored the installation files in.
 If the command exits without a `FATAL` warning, your production control plane
 has initialized.
 
-. Delete the bootstrap resources:
+. To remove the bootstrap instance group from the backend services' backends, run the following commands:
```

Comment:
> backend services' backends

Clearly, I am not an expert (or know anything in this realm). But, thst sounded odd to me. Someone who knows the material might understand this perfectly. Just wanted to point it out.